completion/available/gradle.completion.bash

```diff
@@ -179,10 +185,10 @@ __gradle-generate-tasks-cache() {
 	# Run gradle to retrieve possible tasks and cache.
 	# Reuse Gradle Daemon if IDLE but don't start a new one.
 	local gradle_tasks_output
-	if [[ ! -z "$($gradle_cmd --status 2> /dev/null | grep IDLE)" ]]; then
-		gradle_tasks_output="$($gradle_cmd -b $gradle_build_file --daemon -q tasks --all)"
+	if $gradle_cmd --status 2> /dev/null | grep -q IDLE; then
+		gradle_tasks_output="$($gradle_cmd -b "$gradle_build_file" --daemon -q tasks --all)"
```

Comment:
```suggestion
	if "$gradle_cmd" --status 2> /dev/null | grep -q IDLE; then
		gradle_tasks_output="$("$gradle_cmd" -b "$gradle_build_file" --daemon -q tasks --all)"
```

Reply:
nope, because the command is an exec name and parameters, and quoting it will kill the code.

Reply:
NM, my bad, I confused it with another bit of code. this can+should be quoted. Thanks!